plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectionCreationTest.java

```diff
@@ -34,21 +33,20 @@
 import static java.util.Objects.requireNonNull;
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Test(singleThreaded = true) // this class is stateful, see fields
```

Comment:
what enforces the single-threaded-ness after this change?

Reply:
Nevermind, Junit is single-threaded by default.
However it's still a good idea to explicitly enforce these tests to be non-concurrent because JUnit can be made parallel globally and these tests will then become flaky or outright fail and since the comments are lost whoever debugs it will waste time rediscovering what was already documented.